src/components/Explore/interfaces.ts

```diff
@@ -31,7 +31,7 @@ export enum ExploreMetric {
   VACCINATIONS_ADDITIONAL_DOSE,
   ICU_USED,
   POSITIVITY_RATE,
-  HOSPITALIZATIONS_PER_100K,
+  ADMISSIONS_PER_100K,
```

Comment:
Hrm.  I think we over-renamed here.  ExploreMetric.HOSPITALIZATIONS_PER_100K should not have been affected.